ee/clickhouse/views/test/test_clickhouse_experiments.py

```diff
@@ -232,17 +232,6 @@ def test_create_multivariate_experiment(self):
         id = response.json()["id"]
         end_date = "2021-12-10T00:00"
 
-        # Now try updating FF
-        response = self.client.patch(
-            f"/api/projects/{self.team.id}/experiments/{id}",
-            {
-                "description": "Bazinga",
-                "parameters": {"feature_flag_variants": [{"key": "control", "name": "X", "rollout_percentage": 100}]},
```

Comment:
Are we allowing or not allowing users to be able to edit the "test groups"? @neilkakkar

Reply:
Not allowing for now! (i.e. can't change variants / ff key)

But if you feel its easier, I don't mind deleting / updating existing FF with any new information that comes in^^

Reply:
Hmm, I think you can focus on the other PR, I'll take this one over! Tried updating an experiment on cloud, indeed a pain, will allow updating groups and remove existing FF info!